src/jdk.internal.le/share/classes/jdk/internal/org/jline/utils/OSUtils.java

```diff
@@ -102,4 +103,13 @@ public class OSUtils {
         TEST_COMMAND = test;
     }
 
+    private static boolean isTestCommandValid(String command) {
+        try {
+            Process p = new ProcessBuilder(command, "-z", "").inheritIO().start();
```

Comment:
is there a reason why you chose to spin up a process here ? Would a test for an executable file be sufficient ?